src/TimeSliceGroup.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes, Component } from 'react'
+import TimeSlice from './TimeSlice.jsx'
+import date from './utils/dates.js'
+
+export default class TimeSliceGroup extends Component {
+  static propTypes = {
+    slices: PropTypes.oneOf([2,3,4,5]),
+    size: PropTypes.number.isRequired,
```

Comment:
`size` might be better off having a more descriptive name. right now its a bit confusing as to what it is. maybe `totalMinutes` or something? 

It looks tho like `size` is never used as is. TimeGutter does `step * slices` and its then pulled back out as `size / slices` here. Maybe a better prop is `minutesPerSlice` instead?